troposphere/static/js/components/admin/Emulate.jsx

```diff
@@ -0,0 +1,41 @@
+import $ from "jquery";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import Glyphicon from 'components/common/Glyphicon';
+import Tooltip from "components/common/ui/Tooltip";
```

Comment:
It's used in other places for hover behavior (can't recall its api), and it prevents us from having to write manual DOM manipulation like below.